satcom-modem-interface/satcom-modem-interface.ino

```diff
@@ -68,31 +68,31 @@ void setup()
   #if SDCARD_ENABLE_LED
   digitalWrite(SDCardActivityLEDPin, HIGH);
   #endif
-  Serial.print("Initializing SD card interface...");
+  Serial.print(F("Initializing SD card interface..."));
   while (digitalRead(SDCardDetectPin) == LOW) {
-    Serial.println("SD card not inserted. Waiting.");
+    Serial.println(F("SD card not inserted. Waiting."));
     blinkError(2); 
     delay(1000);
   }
   while (!SD.begin(SDCardCSPin)) {
-    Serial.println("Error initializing SD card interface. Retrying.");
+    Serial.println(F("Error initializing SD card interface. Retrying."));
     blinkError(2); 
     delay(1000);
   }
 
   #if SDCARD_ENABLE_LED
   digitalWrite(SDCardActivityLEDPin, LOW);
   #endif
-  Serial.println("success");
+  Serial.println(F("success"));
 
   IridiumSerial.begin(19200); // Start the serial port connected to the satellite modem
 
   // Begin satellite modem operation
   Serial.print(F("Starting modem..."));
-  int result = modem.begin();
+  int result = 5; // modem.begin();
```

Comment:
This likely wasn't meant to be a committed change.

Reply:
Oof you’re right. Good catch.